ChangeLog.md

```diff
@@ -1,5 +1,23 @@
 # ChangeLog
 
+## [1.0.5] - 2024-10-16
+
+### Added
+- wm.time_stopped, wm.set_play_count, serverParams.goal_area_width/length
+
+### Fixed
+-
+
+### Changed
+- 
+
+### Engineers
```

Comment:
Please don't include Nader and me in the changelog for this update, as we didn't contribute to it.